contracts/plugins/trading/DutchTrade.sol

```diff
@@ -223,13 +284,15 @@ contract DutchTrade is ITrade {
         returns (uint256 soldAmt, uint256 boughtAmt)
     {
         require(msg.sender == address(origin), "only origin can settle");
+        if (bidder == address(0)) {
```

Comment:
these 3 lines can be simplified to `require(bidder != address(0) || block.number > endBlock, "auction not over");`